packages_rs/nextclade-web/src/components/Common/BrowserWarning.tsx

```diff
@@ -21,20 +26,37 @@ export function BrowserWarning() {
     }
 
     const { name, version } = browser.getBrowser()
-    const nameAndVersion = [name, version].filter(notUndefinedOrNull).join(' ')
-
-    return t(
-      `This browser version (${nameAndVersion}) is not supported. Nextclade works best in the latest version of Chrome or Firefox.`,
-    )
-  }, [t])
+    return [name, version].filter(notUndefinedOrNull).join(' ')
+  }, [])
 
-  if (!warningText) {
+  if (!nameAndVersion) {
     return null
   }
 
   return (
-    <UncontrolledAlert color="warning" className="text-center m-0">
-      {warningText}
-    </UncontrolledAlert>
+    <Modal centered isOpen={isOpen} backdrop="static" toggle={dismiss} fade={false} size="lg">
+      <ModalHeader toggle={dismiss} tag="div">
+        <h4 className="text-center text-danger">{t('Unsupported browser')}</h4>
```

Comment:
```suggestion
        <h4 className="text-center text-danger">{t('Unsupported browser: Please use Chrome or Firefox')}</h4>
```

Reply:
This will look very ugly on mobile and also repeats what's in the body.

Reply:
And this is the first thing all iOS users will see when they navigate from Twitter.

Reply:
Hmm, right - we should add that iOS does not work at all, even with Firefox/Chrome